test/unit/index.test.js

```diff
@@ -214,16 +214,16 @@ describe('Protractor Flake', () => {
     })
 
     it('uses protractorRetryConfig file for spawned protractor process only after first attempt', () => {
-      protractorFlake({protractorRetryConfig: __dirname + '/support/protractor.flake.config.js'})
-      expect(spawnStub).to.have.been.calledWith('node', [pathToProtractor(), '--params.flake.iteration', 1])
+      protractorFlake({protractorArgs: [resolve(__dirname + '/support/protractor.flake.config.js')], protractorRetryConfig: resolve(__dirname + '/support/protractor.flake.retryConfig.js')})
```

Comment:
Since this is a unit test we could avoid creating a real file and just pass a path, what do you think?

Reply:
You're right.